app/src/behaviors/behavior_hold_tap.c

```diff
@@ -90,6 +92,8 @@ struct last_tapped {
 
 struct last_tapped last_tapped;
 
+bool is_key_held[ZMK_KEYMAP_LEN] = {0};
```

Comment:
maybe rename to `key_position_pressed`?

Reply:
how about is_key_position_pressed? I do like the "is_" prefix, IMO it helps imply bool